aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/MutationProcessor.java

```diff
@@ -147,6 +147,10 @@ private Completable drainMutationOutbox() {
             HubChannel.DATASTORE,
             HubEvent.create(DataStoreChannelEventName.PUBLISHED_TO_CLOUD, processedMutation)
         );
+        Amplify.Hub.publish(
+            HubChannel.DATASTORE,
+            HubEvent.create(DataStoreChannelEventName.OUTBOX_MUTATION_PROCESSED, processedMutation)
```

Comment:
This is the payload for the event that I see on the design doc. Just make sure it aligns with that. I believe that's what iOS is doing, so you might want to check with @DongQuanRui 

```
{
    "model": "Note",
    "element": {
        "id": "d98f54ff-53dd-44d0-b99f-562e8f58ae3c",
        "content": "content-1594938571139",
        "metadata": null,
        "_version": 1,
        "_lastChangedAt": 1594938571692,
        "_deleted": null
    }
}
```